css/style.css

```diff
@@ -6,14 +6,207 @@ body {
     -webkit-font-smoothing: antialiased;
 }
 
-/**
- * Add your custom styles below
- * 
- * Remember: 
- * - Be organised, use comments and separate your styles into meaningful chunks
- *    for example: General styles, Navigation styles, Hero styles, Footer etc.
- * 
- * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
- */
+ /*Header style*/
+ header{
+    display:flex;
+    justify-content: space-between;
+    align-items: center;
+    margin: 10px 50px;
+}
+ .logo{
+     width:40px;
+     padding-left:100px;
+     min-width: 40px;
+ }
+ nav .big-screen ul{
+     display:flex;
+     flex-wrap:nowrap;
+ }
+ nav .big-screen ul li{
+     list-style-type: none;
+     display: inline;
+     padding:0.5em 1.50em;
+     /* adding calc(), which makes minimum 1em for fontsize,even the device width is very small */
+     font-size:calc(1em + 0.5vw);  
+    
+ }
+ nav .big-screen ul li a{
+    text-decoration: none;
+   
+ }
+ nav .big-screen ul li a:hover{
+    color:orangered;
+ }
+/*Main-hero style*/
+.hero{
+    text-align: center;
+    background-image: url('../img/first-background.jpg');
+    height: 700px;
+    background-repeat: no-repeat;
+    background-position: center;
+    background-size: cover;
+    color:white;
+    display:flex;
+    flex-direction: column;
+    justify-content: center;    
+}
+.hero-headings-bigsize{
+    font-size:calc(2em + 1.5vw);
+}
+.hero-headings-smallsize{
+    font-size:calc(1em + 1vw);
+}
+.hero-button button{
+    background-color: orangered; 
+    height: 3em;   
+    border: none;
+    border-radius: 5%;
+    font-size: calc(0.75em + 1vw);
+    padding: 5px 10px;
+    margin-top: 30px;
+}
+/*Main-Container style*/
+.container{
+    margin-top:75px ;
+}
+ 
+.container-heading p{
+    font-size:calc(1.5em + 1vw);
+    text-align: center;
+}
+.container-icons{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-wrap: wrap;
+    
+}
+.container-icons-wrapper{
+    display:flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+}
+.container-icons-wrapper img{
+    width: 7.5em;
+    margin:2em;    
+}
+.container-icons-wrapper p{
+    font-size: 3em;
+    font-size:calc(1em + 0.75vw);
+    margin:2em;
+    text-align:center;
+  }
+
+
+/*Footer style*/
+.footer-icons img{
+width:1.5em;
+margin-left:10px;
+margin-right:10px;
+}
+footer{
+   display:flex;
+   flex-direction: column;
+   justify-content: center;
+   align-items: center;
+ }
+.footer-heading p, .footer-copyright p{
+    font-size: 1.5em;
+    font-size:calc(0.8em + 0.5vw);
+}
+/* adding media query-inorder not to show navbar for big screens,make visible the mobile nav bar and changing background image height */
+@media screen and (max-width:1500px){
+    nav .big-screen{
+        display: none; 
+    }
+
+    .mobile-screen {
+        display:block;
+      }
```

Comment:
I notice if you open this mobile nav, the main image gets pushed down below it. To avoid this, you need to take the mobile nav 'out of the flow of the page' using `position: absolute;` so it sits on top of the image and doesn't disturb the natural flow of the page.

Reply:
Thank you very much for advising to use position:absolute and explaining now I understand it very clearly. I changed it as you advise and work perfectly. 
.topnav 
{
        overflow: hidden;
        background-color: rgb(94, 91, 91);
        position: absolute;
        display:none;
        min-width: 100vw;
        right:0px;
        top:65px;
       
      }